notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+    "```bash\n",
+    "curl https://raw.githubusercontent.com/halturin/multitail/master/multitail.conf > ~/.multitailrc\n",
+    "```\n",
+    "\n",
+    "### Install Apache Druid\n",
+    "\n",
+    "Run the following to create a dedicated folder for learn-druid in your home directory:\n",
+    "\n",
+    "```bash\n",
+    "cd ~ ; mkdir learn-druid-local\n",
+    "cd learn-druid-local\n",
+    "```\n",
+    "\n",
+    "Now pull and decompress a compatible version of Druid.\n",
+    "\n",
+    "```bash\n",
+    "wget https://dlcdn.apache.org/druid/29.0.0/apache-druid-29.0.0-bin.tar.gz\n",
+    "tar -xzf apache-druid-29.0.0-bin.tar.gz\n",
+    "```\n",
+    "\n",
+    "Use the following commands to rename the folder.\n",
+    "\n",
+    "```bash\n",
+    "mv apache-druid-29.0.0 apache-druid\n",
+    "cd apache-druid\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d07feb7-92dd-42a9-ad6c-c89d5fdad8d5",
+   "metadata": {},
+   "source": [
+    "# Review the log file configuration\n",
+    "\n",
+    "The log file configuration is set in the `log4j2.xml` alongside Druid's own configuration files.\n",
+    "\n",
+    "Remembering that Druid has multiple configuration file locations out-of-the-box, run this command to view the `auto` configuration file for logs that's used by the `learn-druid` script:\n",
+    "\n",
+    "```bash\n",
+    "more ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Within the [`Configuration`](https://logging.apache.org/log4j/log4j-2.4/manual/configuration.html#ConfigurationSyntax) section you will see:\n",
+    "\n",
+    "* [`Properties`](https://logging.apache.org/log4j/2.x/manual/configuration.html#PropertySubstitution) provide key/values pairs that may be used throughout the configuration file.\n",
+    "* [`Appenders`](https://logging.apache.org/log4j/2.x/manual/appenders.html) designate the format of log messages and determine the target for the messages.\n",
+    "* [`Loggers`](https://logging.apache.org/log4j/2.x/manual/configuration.html#Loggers) filter the log messages and dispense them to Appenders. Loggers can filter messages based on the Java package and/or class and by message priority."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca62693b-50f4-4831-a092-dbc8e06d8191",
+   "metadata": {},
+   "source": [
+    "## Properties\n",
+    "\n",
+    "In Druid, `Properties` are leveraged to set a location for all logs, giving Druid the location for log files at start-up.\n",
+    "\n",
+    "By default, this location is a \"log\" folder at the root of your Druid installation, but [can be over-ridden]((https://druid.apache.org/docs/latest/configuration/logging#log-directory) by using the `DRUID_LOG_DIR` system variable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bec19737-2773-4c66-a05e-d0b1a707c4c8",
+   "metadata": {},
+   "source": [
+    "## Appenders\n",
+    "\n",
+    "There are two `appenders`:\n",
```

Comment:
Where? In the Configuration element? Maybe change this line to something like "The Configuration element contains the following appenders:"